power_plant_import/racsv.py

```diff
@@ -0,0 +1,335 @@
+                if cell.value == 'Power Plant Name':
+                    plant_number = cell_number
+                elif cell.value == 'Region':
+                    region_number = cell_number
+                elif cell.value == 'Country':
+                    country_number = cell_number
+                elif cell.value == 'Status':
+                    status_number = cell_number
+                elif cell.value == 'Month Online':
+                    month_number = cell_number
+                elif cell.value == 'Year Online':
+                    year_number = cell_number
+                elif cell.value == 'Decommissioning Year':
+                    deco_year_number = cell_number
+                elif cell.value == 'Owner':
+                    owner_number = cell_number
+                elif cell.value == 'Owner Stake':
+                    owner_stake_number = cell_number
+                elif cell.value == 'Plant Capacity (MW)':
+                    plant_capacity_number = cell_number
+                elif cell.value == 'Total Capacity (MW)':
+                    total_capacity_number = cell_number
+                elif cell.value == 'Active Capacity (MW)':
+                    active_capacity_number = cell_number
+                elif cell.value == 'Pipeline Capacity (MW)':
+                    pipeline_capacity_number = cell_number
+                elif cell.value == 'Discontinued Capacity (MW)':
+                    discon_capacity_number = cell_number
+                elif cell.value == 'Average Output':
+                    average_output_number = cell_number
+                elif cell.value == 'CO2 Emissions (Tonnes per annum)':
+                    co2_number = cell_number
+                elif cell.value == 'Subsidiary Asset Name':
+                    project_name_value = cell_number
+                elif cell.value == 'Capex USD':
+                    capex_number = cell_number
+                elif cell.value == 'Latitude':
+                    latitude_number = cell_number
+                elif cell.value == 'Longitude':
+                    longitude_number = cell_number
+                elif cell.value == 'EPC Contractor':
+                    contractor_one_number = cell_number
+                elif cell.value == 'Operator':
+                    operator_number = cell_number
+                elif cell.value == 'Asset Notes':
+                    asset_number = cell_number
+                
+            writer.writerow(titles)
+
+        
+        # If the country name in country keys, then write the row.
+        if row[country_number].value in countries.keys():
+            # if row[cell_number].font.italic == False:
+            # This will skip the headers
+            if row_number != 0:
+                # At this point we are iterating inside of the titles list that has the headers from the matrix sheet
+                for cell_number, cell in enumerate(titles):
```

Comment:
Switching from `csv.writer` to `csv.DictWriter` and tracking original header -> output header as a dictionary would probably simplify this large `if/elif` block.